src/admin/services/Utils.ts

```diff
@@ -35,34 +36,51 @@ export function getPaginationParams (search: string): Pagination {
 
 	return {
 		page: parseInt(query.get('page')) || 1,
-		limit: parseInt(query.get('limit')),
+		limit: parseInt(query.get('limit')) || defaultLimit,
 		sort: query.get('sort'),
 		sortOrder: parseInt(query.get('sortOrder')),
 	};
 }
 
-export function getFormattedPagination(page?, limit?, sort?, sortOrder?) {
-	let result: string = '';
-	const pagination: string[] = [];
+export function getFormattedPagination(
+	page?: number,
+	limit?: number,
+	sort?: string,
+	sortOrder?: number
+) {
+	const query = new URLSearchParams();
+	if (page && page > 0) {
+		query.append('page', page.toString());
+	}
+	if (typeof limit === 'number') {
+		query.append('limit', limit.toString());
+	}
+	if (sort && typeof sortOrder !== 'undefined') {
+		query.append('sort', sort);
+		query.append('sortOrder', sortOrder.toString());
+	}
 
-	if (sort || page || limit) {
-		if (page) {
-			pagination.push(`page=${page}`);
-		}
-		if (limit) {
-			pagination.push(`limit=${limit}`);
-		}
-		if (sort) {
-			pagination.push(`sort=${sort}`);
+	return query.toString() ? `?${query.toString()}` : '';
+}
 
-			if (sortOrder) {
-				pagination.push(`sortOrder=${sortOrder}`);
+// Returns a function, that, as long as it continues to be invoked, will not
+// be triggered. The function will be called after it stops being called for
+// N milliseconds. If `immediate` is passed, trigger the function on the
+// leading edge, instead of the trailing.
+export function debounce(func, wait, immediate) {
```

Comment:
You have lodash in project. You don`t need custom functiom i think...
https://lodash.com/docs/#debounce

Reply:
It was for front end, where we don't connect lodash because of it's high weight, but at the end I don't use it at all, so I just would remove it, thank you for remind me about it